### ISSUE TITLE
Overscan Not Work

### ISSUE THREAD

**Issue by ToniBC** [libretro/mupen64plus-libretro-nx#264]
The Overscan options that previously worked to reduce or remove vertical black stripes stopped working. Now it does not matter what value you put, it does nothing.

![Overscan Not Work](https://user-images.githubusercontent.com/45365478/94679490-7adbe400-0320-11eb-9123-77798167973f.png)

Reset the core configuration and Retroarch.cfg, but nothing, those options no longer work.

Something changed, because using the Parallel plugin doesn't work either.

All this under Windows 10 x64 and Vulkan.

Thank you.

**Comment by m4xw** [comment]
![image](https://user-images.githubusercontent.com/13141469/94683841-74049f80-0327-11eb-882c-e62d4b2f2137.png)
Fresh built develop branch works just fine also gliden wasnt changed
[mupen64plus_next_libretro.zip](https://github.com/libretro/mupen64plus-libretro-nx/files/5305339/mupen64plus_next_libretro.zip)

**Comment by ToniBC** [comment]
Well I'm sorry, but no, it doesn't work, Windows is up to date, the nvidia drivers in their latest version 456.55 from 2 days ago.

Tested in Retroarch clean both Stable and Nightly, just download the core and load, the overscan does not work and before if, and with the drivers parallel neither load, Retroarch closes.

They notified me of that error, because they were trying to adjust the image and it was not working for them and I checked it and it does not work.

I even tried that version you left, which is the same, but to be tested.

Tried with GL and Vulkan and nothing.
